src/providers/Theme.vue

```diff
@@ -66,6 +70,7 @@ export default defineComponent({
     --color-1: #ffffff;
     --color-1-contrast: #ffffff;
     --color-2: #573a90; // Web / Tile / Button / Secondary
+    --color-3: #ffffff; // Main Widget Button
     --bg-1: linear-gradient(
       90deg,
       #b83990 18.24%,
```

Comment:
--color-3 will be used only for main widget button doing forward as it doesn't seem to have a relation to the other button colors. Checkout design 👆 
![image](https://user-images.githubusercontent.com/63985172/135382612-176909fc-3939-4712-9c9d-fc4a434e1e2d.png)

Reply:
If the theme colours grow any larger it's a good idea to add even more comments per colour for where it's used 🙏